app/models/crop.rb

```diff
@@ -206,7 +247,22 @@ def add_alternate_names_from_csv(alternate_names)
   # searches for crops whose names match the string given
   # just uses SQL LIKE for now, but can be made fancier later
   def self.search(query)
-    where("name ILIKE ?", "%#{query}%")
+    search_str = query.nil? ? "" : "#{query.downcase}"
```

Comment:
quoting was needed in the hash, so I'm moving the quite in the hash to make it clearer.